assets/src/components/UserSettingSnackbar.js

```diff
@@ -2,6 +2,11 @@ import React, { useState, useEffect } from 'react'
 import Snackbar from '@material-ui/core/Snackbar'
 import IconButton from '@material-ui/core/IconButton'
 import CloseIcon from '@material-ui/icons/Close'
+import Slide from '@material-ui/core/Slide'
+
+function SlideTransition(props) {
```

Comment:
Add a space so it's `SlideTransition (props)`.

Reply:
fixed